redbot/core/utils/_internal_utils.py

```diff
@@ -282,3 +295,14 @@ async def preprocessor(bot: Red, destination: discord.abc.Messageable, content:
         return content.replace("[p]", prefix)
 
     await send_to_owners_with_preprocessor(bot, content, content_preprocessor=preprocessor)
+
+
+def expected_version(current: str, expected: str) -> bool:
+    re_match = re.match(version_req_re, expected)
+    specifier, expected = re_match.group(1), re_match.group(2)
+    if specifier is None:
+        specifier = ">="
+    try:
+        return operator_lookup[specifier](LooseVersion(current), LooseVersion(expected))
+    except TypeError:  # Invalid Expected version
+        return False
```

Comment:
Now that I look at this again... It would be nice if this supported specifier sets, i.e. `">=3.6.2,<3.8"` - we have used such specifier sets in the past on `python_version` and we can't know if we won't use use them in future. We could use `pkg_resources` for that:
```suggestion
def expected_version(current: str, expected: str) -> bool:
    # `pkg_resources` needs a regular requirement string, so "x" serves as requirement's name here
    return current in pkg_resources.Requirement.parse(f"x{expected}")
```